src/components/ProductItem/ProductItem.tsx

```diff
@@ -333,78 +361,82 @@ export const ProductItem: FunctionComponent<ProductProps> = ({
                 className={`max-h-[45rem] w-full object-cover object-center lg:w-full`}
               />
             )}
-            {<div className="absolute left-0 right-0 bottom-0 p-xs">
-          {screenSize.mobile && <AddToCartButton onClick={handleAddToCart} />}
-          {isHovering && screenSize.desktop && (
-            <AddToCartButton onClick={handleAddToCart} />
-          )}
-        </div>}
+            <div className="add-to-cart-overlay absolute left-0 right-0 bottom-0 p-xsmall h-[56px]">
+              {!screenSize.desktop && !showSizes && <AddToCartButton onClick={handleAddToCart} />}
+              {isHovering && screenSize.desktop && !showSizes && (
+                <AddToCartButton onClick={handleAddToCart} />
+              )}
+              {showSizes && productView?.options?.map((swatches) => {
```

Comment:
Will add logic to hide size swatches on onMouseLeave
The decision is still pending on design side

Reply:
Made changes